### ISSUE TITLE
Releasing a room is not synchronised back to MS365 account. Reservation works fine.

### ISSUE THREAD

**Issue by pmhausen** [synyx/MeeRoo#124]
Hi folks,

what details would you need to help in debugging? We are running the latest release.

Kind regards,
Patrick

**Comment by maxdobler** [comment]
Hi Patrick,

we never tested the app with Outlook calendars. Therefore it is a bit hard for me to think of any possible solution.
What we know is, that there were different implementations when it comes to recurring events in Google Calendar events and CalDav based calendars.

Did the problem occur when releasing a recurring event?

I will try to set up a test scenario with Outlook calendars in the next days and try to dig into this problem.

Kind regards,
Max

**Comment by pmhausen** [comment]
No, no recurring event. We simply set up an MS365 ("Exchange") account in the gmail app, then used that in Meeroo.
Tap "30 minutes", event in calendar is created, tap "release" - nothing. When one deletes the event in the MS365 web UI, the tablet correctly turns green and "available".
